config/initializers/good_job.rb

```diff
@@ -88,5 +88,9 @@
       cron: "every day at 08:00 Europe/Paris",
       class: "CronJob::NotifySmsFactorLowCredits",
     },
+    send_mattermost_notifications_for_zammad_tickets: {
+      cron: "every day at 10:15 Europe/Paris",
```

Comment:
```suggestion
      cron: "every weekday at 10:15 Europe/Paris",
```
Je pense qu’on ne peut envoyer ça que la semaine

```
lapin(dev)> Fugit::Nat.parse("every weekday at 10:15 Europe/Paris").to_cron_s
=> "15 10 * * 1,2,3,4,5 Europe/Paris"
```

Alternative : `15 10 * * 1-5 Europe/Paris`